packages/eslint-config-typescript/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@mollie/eslint-config-typescript",
+  "version": "0.0.1-alpha.4",
+  "description": "Mollie's ESLint configuration preset for TypeScript.",
+  "main": "index.js",
+  "author": "Mollie B.V. <info@mollie.com>",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "eslint": ">= 3"
+  },
+  "dependencies": {
+    "@mollie/eslint-config-base": "^0.0.1-alpha.3",
+    "@typescript-eslint/eslint-plugin": "^1.3.0",
+    "@typescript-eslint/parser": "^1.3.0",
+    "typescript": "^3.3.3"
```

Comment:
Do we need this as a dependency? Or can it be a peerDepencency?

Reply:
It looks like a peerDependency of `typescript >= 3` will do. I'll change it.

Reply:
Nope, `typescript >= 3.2.1`. The ESLint plugin requires a relatively high version.

Reply:
And `eslint >= 5.12.1` it seems.